### ISSUE TITLE
New superparents for all subclasses of bfo 'temporal region' 

### ISSUE THREAD

**Issue by mark-jensen** [CommonCoreOntology/CommonCoreOntologies#213]
BFO2020 introduces two new terms:
- 'temporal interval' =def (Elucidation) A temporal interval is a one-dimensional temporal region that is continuous, thus without gaps or breaks
- 'temporal instant' =def (Elucidation) A temporal instant is a zero-dimensional temporal region that has no proper temporal part

From what I can tell all of the CCO classes which were previously subclasses of '[one|zero]-dimensional temporal region' should be relocated under either one of these.

Any reason not do this?

@johnbeve @APCox @neilotte @cameronmore 

<img width="709" alt="Screenshot 2024-01-23 at 9 10 51 AM" src="https://github.com/CommonCoreOntology/CommonCoreOntologies/assets/11448917/7ebae44c-fba4-4809-8f0e-ddfb1a6667ee">

**Comment by johnbeve** [comment]
@mark-jensen @cameronmore @APCox 'one-dimensional temporal region' above is defined to prevent gaps but the updated definition allows for gaps, hence the introduction of the 'temporal interval' subclass. This is to say I think most of the above subclasses would fall under temporal interval, though I have a reservation: 

Do the multi- defined classes allow for temporal gaps? The definitions suggest they might, e.g. multi-day temporal region =def A One-Dimensional Temporal Region that is measured in Days and spans at least one Day. 

I read this as permitting gaps, which might be useful if one wanted to model, say, all the holidays in a calendar year, all the pay days in a month, etc. If so, then it would fall under the new definition of 'one-dimensional temporal region' but not under 'temporal interval'.

**Comment by mark-jensen** [comment]
@johnbeve Good catch. I had the same reservation. It seems quite plausible that they allow for discontinuous regions. But, in discussion with Alex and Cameron a few questions were raised which has us thinking these "multi-" terms should be deleted.

How many hours did I work last week? Presumably my working processes occurred on two or more disconnected intervals, each of which has a duration value associated with it. And the answer to the question is to add those values, say = "46".  Is there a good reason to go further and assert that the "46" value is linked to an ICE that is about an instance of a one-d TR that contains all those other continuous intervals? Skeptical, but ok, maybe. So, how does use of a Multi-Hour TR help over just using a plain one-d TR? Why is a multi-Day interval not also a multi-Hour?

My guess is that these are legacy from earlier days in CCO when terms for Week, Hour, Year, etc, were not in the CCO and perhaps these were a way of smuggling in a quasi- temporal measurement unit via the class. Note that we have measurement units for most of these in CCO now.  I think at best many of these subs of one-d TR are defined classes based on the measurement unit of an ICE about the TR. Creating specializations of temporal regions based on the fiat way humans record durations of processes seems dubious to me. Is an hour any different than a meter or a peck? Would it make sense to also introduce subtypes of 'bfo fiat line' based on all of the common measurement units we use for lengths?

Perhaps the best solution for now is to leave the multis where they are, move the rest, and start a new issue to explore refactoring CCO subs of TRs, seeing which should stay vs. go, and how to redefine those that stay.

**Comment by swartik** [comment]
@johnbeve @mark-jensen Do you really use "multi-" that way? I grant John's point that modeling all holidays could be useful. But I would never describe that as a "multi-day temporal region". It's just not how I grew up understanding the term. I would characterize "All Holidays" as a specialization of one-dimensional temporal region whose temporal parts are constrained to be specific days. (Individuals would correspond to periods during which certain holidays were recognized. The latest version of all Federal holidays would include Juneteenth.)

Anyway, that's just my opinion. I have occasionally used the Multi-* Temporal Interval classes with the implicit, and perhaps incorrect, understanding that the *'s were contiguous. Whatever the resolution, it needs to be stated.

**Comment by johnbeve** [comment]
@swartik Good points; common use of "multi-day" does seem to suggest contiguity. I agree whatever direction is pursued, it should be reflected in the definition. If anything because common use of "multi-day" aside, there are many examples of "multi-" expressions that don't imply contiguity, e.g. multicellular, multiphase, multivitamin. Also, I doubt many folks have firm natural language intuitions about "multi-day temporal interval", since it's a technical expression. 

@mark-jensen Stepping back, I think you're on to something; isn't "hour" a unit of measurement? Same for multi-hour, etc.

**Comment by mark-jensen** [comment]
This issue was primarily addressed in 010c9a2bc0c78df3e442babaebdcfd00b80d386d wherein most subtypes of 'temporal region' were moved under 'temporal interval' or 'temporal instant'. These charges are part of v1.5. 

The remaining points about the need for subtypes of the form `multi-X temporal interval` and whether some of the subtypes of 'temporal region' are better, at least primarily, represented as `units of measurement`, e.g., second, hour, etc., will be taken up in a discussion.
